ONSdigital/eq-author
# Fix problem with active state on nav and main tabs

### What is the context of this PR?

[Trello card](https://trello.com/c/3E4SGGrs/345-sidebar-and-top-tabs-not-highlighting-the-selected-page)

### How to review 

- navigate app via side navigation and check active page is highlighted
- navigate between design/routing tabs and check active tab is highlighted
